qlib/workflow/__init__.py

```diff
@@ -34,8 +37,13 @@ def start(self, experiment_name=None, recorder_name=None):
             name of the experiment one wants to start.
         recorder_name : str
             name of the recorder under the experiment one wants to start.
+        uri : str
```

Comment:
Please fix some typos
are/is 
user/users
occur/occurs

Reply:
Fixed~